lib/lokka/handlers/file_upload_handler.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
```

Comment:
Added FileUploadHandler that deals File Upload to Amazon S3 on admin console (XHR)